docs/root/version_history/current.rst

```diff
@@ -74,6 +74,7 @@ New Features
 * listener: added ability to change an existing listener's address.
 * metric service: added support for sending metric tags as labels. This can be enabled by setting the :ref:`emit_tags_as_labels <envoy_v3_api_field_config.metrics.v3.MetricsServiceConfig.emit_tags_as_labels>` field to true.
 * tcp: added support for :ref:`preconnecting <v1.18.0:envoy_v3_api_msg_config.cluster.v3.Cluster.PreconnectPolicy>`. Preconnecting is off by default, but recommended for clusters serving latency-sensitive traffic.
+* tls: allow dual ECDSA/RSA certs via SDS. This was previously only supported via static file-based certificate configuration.
```

Comment:
That's not true; it was possible to do inline ECDS/RSA certs via TLS contexts included directly in `Listener` or `Cluster` transport socket configuration. Suggest rephrasing to reflect this.

Reply:
Ah, I see. That's what I intended to say, but the phrasing came out misleading. Will fix.